src/components/typography/typography.mdx

```diff
@@ -29,15 +29,27 @@ It's possible to change the text colour and background, using the color props. S
 
 It's possible to change the text position, using the spacing props. See the prop tables below for more information.
 
-It's also possible to create ordered and unordered lists using the `List`, `ListItem` helpers. These components are syntactic sugar around the `Typography` component and accept the same props.
-
 <Canvas of={TypographyStories.VariantsStory} />
 
+## Lists
```

Comment:
**praise**: great idea to make the list functionality more explicit in the docs 👍🏼